src/slg/textures/math/rounding.cpp

```diff
@@ -49,10 +49,12 @@ Properties RoundingTexture::ToProperties(const ImageMapCache &imgMapCache, const
 float RoundingTexture::round(float value, float increment) const {
     if(value == increment) {
         return value;
+    } else if(increment == 0) {
+        return 0.f;
```

Comment:
Hm, if increment is 0, doesn't that mean that no rounding should happen?
So just `return value` in that case?

Reply:
That would make sense, yeah. An increment of 0 defeats the purpose of using the texture either way.